JenkinsOKEfile

```diff
@@ -0,0 +1,594 @@
+                                                sh "chmod 666 ${WORKSPACE}/terraform/oci_api_key.pem"
+
+                        }
+
+                        sh '''
+
+                        export OCI_CLI_SUPPRESS_FILE_PERMISSIONS_WARNING=True
+
+                        echo 'Set env vars needed for integration tests...'
+
+
+                        export OKE_CLUSTER="true"
+                        export FSS_DIR="/oketest"
+
+                        export VCN_CIDR_PREFIX=${VCN_CIDR_PREFIX}
+
+                        export OKE_CLUSTER_NAME=${OKE_CLUSTER_NAME}
+                        export NODE_SHAPE=${NODE_SHAPE}
+                        export LINUX_VER=${LINUX_VER}
+
+                        export OCI_PROP_FILE=${OCI_PROP_FILE}
+
+                        #export IMAGE_TAG_WEBLOGIC="12.2.1.4"
+                        #export IMAGE_TAG_FMWINFRA="12.2.1.4"
+
+                        #################
+                        echo "Generating property file oci.prop for terraform scripts"
+
+                        cat > $OCI_PROP_FILE <<EOF
+
+                        user.ocid=${USER_OCID}
+                        okeclustername=${OKE_CLUSTER_NAME}
+                        tfvars.filename=${OKE_CLUSTER_NAME}
+                        region=us-phoenix-1
+                        tenancy.ocid=${TENANCY_OCID}
+                        compartment.ocid=${COMPARTMENT_OCID}
+                        compartment.name=QualityAssurance
+                        ociapi.pubkey.fingerprint=${FINGERPRINT}
+                        ocipk.path=${WORKSPACE}/terraform/oci_api_key.pem
+                        vcn.cidr.prefix=${VCN_CIDR_PREFIX}
+                        vcn.cidr=${VCN_CIDR_PREFIX}.0.0/16
+                        nodepool.shape=${NODE_SHAPE}
+                        nodepool.imagename=${LINUX_VER}
+                        k8s.version=v${KUBE_VERSION}
+                        nodepool.ssh.pubkey=${WKOBUILD_PUBLIC_CERT}
+                        terraform.installdir=${WORKSPACE}/terraform/terraforminstall
+EOF
+
+                        ##################
+
+                        cp -rf ${WORKSPACE}/kubernetes/samples/scripts/terraform/template.tfvars ${WORKSPACE}/terraform/.
+                        mkdir -p ${WORKSPACE}/terraform/terraforminstall
+                        '''
+
+                    }
+                }
+
+                stage('Build WebLogic Kubernetes Operator') {
+                    environment {
+                        DOCKERHUB_USERNAME = credentials("${dockerhub_username_creds}")
+                        DOCKERHUB_PASSWORD = credentials("${dockerhub_password_creds}")
+                    }
+                    steps {
+                        sh 'echo ${DOCKERHUB_PASSWORD} | docker login -u ${DOCKERHUB_USERNAME} --password-stdin'
+                        sh "mvn -DtrimStackTrace=false clean install"
+                    }
+                }
+
+                stage('Run Sonar Analysis') {
```

Comment:
You should not run Sonar on these builds.  Sonar is already running on the PR gate builds, which should be sufficient.